src/pages/distroSettings/tabs/GeneralTab/getFormSchema.ts

```diff
@@ -0,0 +1,106 @@
+import { css } from "@emotion/react";
+import { GetFormSchema } from "components/SpruceForm";
+import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+
+export const getFormSchema = (): ReturnType<GetFormSchema> => ({
+  fields: {},
+  schema: {
+    type: "object" as "object",
+    properties: {
+      distroName: {
+        type: "object" as "object",
+        title: "Identifier",
```

Comment:
[nit] Would maybe remove the title here since it's kinda redundant

```suggestion
        title: "",
```